include/openenclave/internal/syscall_decls.h

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#ifndef _OE_INTERNAL_SYSCALL_DECLS_H
```

Comment:
https://github.com/openenclave/openenclave/blob/master/docs/Community/DevelopmentGuide.md says:

> DON'T use abbreviations unless they are already well-known terms known by users (e.g., "app", "info"), or are already required for use by developers (e.g., "min", "max", "args"). Examples of bad use would be `num_widgets` instead of `widget_count`, and `opt_widgets` instead of `option_widgets` or `optional_widgets`.

Any reason this file should say "decls" instead of "declarations"?  After all, the macro is OE_DECLARE_SYSCALL7 not OE_DECL_SYSCALL...

Reply:
Moved/renamed the file to include/openenclave/internal/syscall/declarations.h
Updated included guard.